### ISSUE TITLE
Mark dependencies or skip

### ISSUE THREAD

**Issue by evnu** [alfert/propcheck#193]
With Elixir v1.11:

````
warning: EEx.compile_file/2 defined in application :eex is used by the current application but the current application does not depend on :eex. To fix this, you must do one of:

  1. If :eex is part of Erlang/Elixir, you must include it under :extra_applications inside "def application" in your mix.exs

  2. If :eex is a dependency, make sure it is listed under "def deps" in your mix.exs

  3. In case you don't want to add a requirement to :eex, you may optionally skip this warning by adding [xref: [exclude: [EEx]]] to your "def project" in mix.exs

  lib/coverex/task.ex:225: Coverex.Task

warning: EEx.function_from_file/4 defined in application :eex is used by the current application but the current application does not depend on :eex. To fix this, you must do one of:

  1. If :eex is part of Erlang/Elixir, you must include it under :extra_applications inside "def application" in your mix.exs

  2. If :eex is a dependency, make sure it is listed under "def deps" in your mix.exs

  3. In case you don't want to add a requirement to :eex, you may optionally skip this warning by adding [xref: [exclude: [EEx]]] to your "def project" in mix.exs

  lib/coverex/task.ex:225: Coverex.Task

warning: :cover.analyse/3 defined in application :tools is used by the current application but the current application does not depend on :tools. To fix this, you must do one of:

  1. If :tools is part of Erlang/Elixir, you must include it under :extra_applications inside "def application" in your mix.exs

  2. If :tools is a dependency, make sure it is listed under "def deps" in your mix.exs

  3. In case you don't want to add a requirement to :tools, you may optionally skip this warning by adding [xref: [exclude: [:cover]]] to your "def project" in mix.exs

Found at 4 locations:
  lib/coverex/source.ex:23: Coverex.Source.analyze_to_html/1
  lib/coverex/source.ex:103: Coverex.Source.cover_per_mod/1
  lib/coverex/task.ex:199: Coverex.Task.coverage_data/1
  lib/coverex/task.ex:203: Coverex.Task.coverage_data/1

warning: :cover.compile_beam_directory/1 defined in application :tools is used by the current application but the current application does not depend on :tools. To fix this, you must do one of:

  1. If :tools is part of Erlang/Elixir, you must include it under :extra_applications inside "def application" in your mix.exs

  2. If :tools is a dependency, make sure it is listed under "def deps" in your mix.exs

  3. In case you don't want to add a requirement to :tools, you may optionally skip this warning by adding [xref: [exclude: [:cover]]] to your "def project" in mix.exs

  lib/coverex/task.ex:24: Coverex.Task.start/2

warning: :cover.modules/0 defined in application :tools is used by the current application but the current application does not depend on :tools. To fix this, you must do one of:

  1. If :tools is part of Erlang/Elixir, you must include it under :extra_applications inside "def application" in your mix.exs

  2. If :tools is a dependency, make sure it is listed under "def deps" in your mix.exs

  3. In case you don't want to add a requirement to :tools, you may optionally skip this warning by adding [xref: [exclude: [:cover]]] to your "def project" in mix.exs

Found at 2 locations:
  lib/coverex/task.ex:38: Coverex.Task.start/2
  lib/coverex/task.ex:57: Coverex.Task.start/2

warning: :cover.start/0 defined in application :tools is used by the current application but the current application does not depend on :tools. To fix this, you must do one of:

  1. If :tools is part of Erlang/Elixir, you must include it under :extra_applications inside "def application" in your mix.exs

  2. If :tools is a dependency, make sure it is listed under "def deps" in your mix.exs

  3. In case you don't want to add a requirement to :tools, you may optionally skip this warning by adding [xref: [exclude: [:cover]]] to your "def project" in mix.exs

  lib/coverex/task.ex:23: Coverex.Task.start/2
````

**Comment by evnu** [comment]
Ah, this needs to be fixed in [`coverex`](https://github.com/alfert/coverex). Sorry for the noise.
